src/components/book.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+const Book = () => {
+  const [bookState, setBookState] = useState('');
+
+  const Handledelete = () => {
+    setBookState('Book deleted');
+  };
+
+  return (
+    <div>
+      <p>
+        Individual Book State:
+        {bookState}
+      </p>
+      <button type="button" onClick={Handledelete}>Remove</button>
+    </div>
+  );
+};
+
+export default Book;
```

Comment:
- [ ] The book component should take the book data as a prop and render the title and the author as per requirement. Kindly do so.

The following is just an example:

```
import React from 'react';
import PropTypes from 'prop-types';

const BookItem = ({ book }) => {
  const {
    booktitle, author, id, category,
  } = book;
  return (
    <li key={id}>
      <p>
        Book:
        {booktitle}
      </p>
      <p>
        Author:
        {author}
      </p>
      <p>
        Category:
        {category}
      </p>
      <button type="button" itemID={id}>Remove</button>
    </li>
  );
};

BookItem.propTypes = {
  book: PropTypes.shape(
    {
      booktitle: PropTypes.string.isRequired,
      id: PropTypes.string.isRequired,
      author: PropTypes.string.isRequired,
      category: PropTypes.string.isRequired,
    },
  ).isRequired,
};

export default BookItem
```